tests/ut_seattlelibv2_repypp_include.py

```diff
@@ -0,0 +1,28 @@
+"""
+ut_seattlelibv2_repypp_include.py -- -- This script tests correct #include statements
+
+"""
+
+import os
+import tempfile
+import subprocess
+
+def main():
+    #create a temporary_file1 contains:
+    #def foo():
+    # pass
+    temporary_file1 = tempfile.NamedTemporaryFile(prefix='testfile_', suffix='.py', dir= os.path.dirname(os.path.realpath(__file__)), delete=True)
+    temporary_file1.writelines(['def foo():\n','  pass'])
+
+    #create a temporary_file2 contains:
+    #include temporary_file1.py
+    #def bar():
+    # pass
+    temporary_file2 = tempfile.NamedTemporaryFile(prefix='testfile_', suffix='.py', dir= os.path.dirname(os.path.realpath(__file__)), delete=True)
+    temporary_file2.writelines(['include ' + temporary_file1.name + '\n','def bar():\n','  pass'])
+
+    subprocess.call(['python', 'repypp.py',temporary_file2.name, 'unittest.py'])
```

Comment:
So the check sums up to calling `repypp`. If it raises no error, then you assume the test passed.

Can we check that the file contents match what we expect? I know `repypp` adds a few things as comments dynamically (file path IIRC), but we should have that the processed file contains everything from the included file, and everything from the including file except for the lines stating `include ...`.